component/alerts/alert.css

```diff
@@ -0,0 +1,57 @@
+.alert_container {
+    border-radius: 0.5rem;
+    background-color: var(--primary-alert-color);
+    font-size: 1.5rem;
+    height: 4rem;
+    padding: 1rem;
+    width: 50rem;
+}
+
+#Alerts .component_box {
```

Comment:
It's better to use class selectors than id selectors.

Reply:
ok I will change.